packages/adelus/example/adelus_driver.cpp

```diff
@@ -168,10 +168,9 @@ int main( int argc, char* argv[] )
 #else
   hipGetDeviceCount ( &gpu_count );
 #endif
-  Kokkos::InitArguments args;
-  args.num_threads = 0;
-  args.num_numa    = 0;
-  args.device_id   = rank%nptile;
+  Kokkos::InitializationSettings args;
+  args.set_num_threads(0);
+  args.set_device_id(rank%nptile);
```

Comment:
@ndellingwood The ```args.device_id``` in the printf statement right below this line should be changed to ```args.get_device_id()```

Reply:
@vqd8a updated with caa9f8078401e2678ce7e60facf8daeae2f3fd43

Reply:
similar to previous comments:
@vqd8a I can update to args.set_num_threads(1); in this PR if you like
@vqd8a I can drop the args.set_device_id(rank%gpu_count); in this PR if you like

Reply:
@ndellingwood please do both. Thanks.

Reply:
Updated with e60b1c0807bc945b72d8b3e12d0e2cd0d75b04bf, thanks @vqd8a @dalg24